### ISSUE TITLE
pulling "shared with me" not working

### ISSUE THREAD

**Issue by leononame** [odeke-em/drive#1089]
I have several folders that are "shared with me" in drive and I did add them to "My Drive" as described in #643 or #1076. 

However, running `drive pull` will download each folder as a flat file with no data (0b) and a `.desktop` sidecar that links to the folder (but the link is broken)

**Comment by HorvathAkosPeter** [comment]
It is such a wonderful tool... and it is now totally unusable. It is the missing feature of the year.

No, the other issues mentioning this simply don't work. I am really sorry. I left the project.

**Comment by gkostkowski** [comment]
I have the same problem, any chances to fix it?

**Comment by leononame** [comment]
No, it seems this project is dead. I don't know if an active fork exists somewhere

**Comment by odeke-em** [comment]
Sorry about that. I believe drive/v2 has been deprecated hence the oddities
with things stopping to work, and drive/v3 requires a whole lot of work.
I’ve been drowning in life and work, trying to build a company and barely
getting any time. Please feel free to use other projects though, and I
appreciate y’all using it in the past, but please go ahead. There is rclone
which is actively maintained. Once I have some free time I’ll send fixes
but otherwise am overwhelmed. Thank you
